### DISCUSSION TITLE
Multiple Slice

### DISCUSSION THREAD

**Discussion by abida-02** [srsran/oran-sc-ric#40] [General]
the oran specification currently supports slice_level_prb.  I want to connect two simple_rc_xapp with two different slice. Is it possible to do this with e2_sm_rc ervice model?

**Comment by pgawlowicz** [comment]
currently it is not supported,
On the xapp side you would need to specify the slice, see here:
https://github.com/srsran/oran-sc-ric/blob/main/xApps/python/lib/e2sm_rc_module.py#L47

But it must also be handled on the srsgnb side. And currently it is not.

**Comment by abida-02** [comment]
I already  enable two slice on gnb side. Now i want to send rc message to the each slice. Is that possible with the oran-sc-ric provided xapp?

**Reply by pgawlowicz** [reply]
currently it is not supported
